pkg/repository/repository.go

```diff
@@ -669,3 +676,74 @@ func (r *repository) transpileFilter(filter filtering.Filter) (*clause.Expr, err
 		filter: filter,
 	}).Transpile()
 }
+
+func (r *repository) ListModelTriggers(ctx context.Context, pageSize int64, page int64, order ordering.OrderBy, modelUID *string, startTimeFrom *time.Time, startTimeTo *time.Time) (modelTriggers []*datamodel.ModelTrigger, totalSize int64, err error) {
+	logger, _ := custom_logger.GetZapLogger(ctx)
+
+	var whereConditions []string
+	var whereArgs []any
+	if modelUID != nil {
```

Comment:
Should we allow for an empty `modelUID` param?

Reply:
no, updated